assets/js/modules/tagmanager/components/settings/SettingsForm.js

```diff
@@ -45,6 +76,16 @@ export default function SettingsForm() {
 				<WebContainerSelect />
 
 				<AMPContainerSelect />
+
+				{ isResolvingWebGetLiveContainerVersion ||
+				isResolvingAMPGetLiveContainerVersion ? (
```

Comment:
With the above change giving the single value for this condition, this can be simplified into `isResolvingLiveContainerVersion && ( ... )` which we generally prefer as a convention over the ternary